docs/nx-cloud/concepts/cache-security.md

```diff
@@ -32,9 +32,14 @@ You can strengthen your workspace security further by revoking all access to the
 
 While you can [specify a token](/ci/recipes/security/access-tokens) with the `nxCloudAccessToken` property in `nx.json`, this is visible to anyone who can view your codebase. A read-write token would give someone who may not even have permission to create a PR the access to add entries to the remote cache, which would then be used on other developer's machines and in CI. We recommend you restrict CI Access Tokens to CI use only and rely on [personal access tokens](/ci/recipes/security/personal-access-tokens) for local development instead.
 
-### Use a Read-Write Token in CI
+### Use Scoped Tokens in CI
 
-If you're in an environment (like an open source project) where you can't trust the contents of a pull request, we recommend restricting the use of a [read-write token](/ci/recipes/security/access-tokens) in CI to just be used on the `main` branch. If you know that everyone who can make a PR is a trusted developer, you can extend that [read-write token](/ci/recipes/security/access-tokens) to also include pull request branches.
+We recommend using a [read-write token](/ci/recipes/security/access-tokens#read-write-access) only for protected branches (branches that don't allow direct push).
+The `read-write` access tokens allow reading from and writing to the shared global cache of your workspace.
```

Comment:
```suggestion 
A `read-write` access token allows reading from and writing to the shared global cache of your workspace.
```

Just for consistency since we talk about `read-only` tokens in singular phrasing below